cms/static/cms/js/modules/cms.modal.js

```diff
@@ -344,9 +344,9 @@
 
                 this.ui.shim.show();
 
-                this.ui.body.on('mousemove.cms', function (e) {
-                    var left = position.left - (initial.pageX - e.pageX);
-                    var top = position.top - (initial.pageY - e.pageY);
+                this.ui.body.attr('touch-action', 'none').on('pointermove.cms', function (e) {
```

Comment:
we still need to find a version to make `touch-action` either validate or getting a `data-touch-action` workaround